cmd/generate/config/rules/atlassian.go

```diff
@@ -11,15 +11,26 @@ func Atlassian() *config.Rule {
 	r := config.Rule{
 		Description: "Detected an Atlassian API token, posing a threat to project management and collaboration tool security and data confidentiality.",
 		RuleID:      "atlassian-api-token",
-		Regex: utils.GenerateSemiGenericRegex([]string{
-			"atlassian", "confluence", "jira"}, utils.AlphaNumeric("24"), true),
+		Regex: utils.MergeRegexps(
+			utils.GenerateSemiGenericRegex(
+				[]string{"atlassian", "confluence", "jira"},
+				utils.AlphaNumeric("24"),
+				true,
+			),
+			utils.GenerateSemiGenericRegex(
```

Comment:
Every token I've found has the prefix `ATATT3xFfGF0`. It seems like this could instead be
```suggestion
			utils.GenerateUniqueTokenRegex(`ATATT3[A-Za-z0-9_\-=]{186}`, false),
```

Reply:
Moreover, they all seem to have the suffix `=[a-fA-F0-9]{8}`. `=` doesn't seem to be valid outside of that.